mobile/Ihc/components/Container.js

```diff
@@ -127,17 +127,10 @@ class Container extends Component<{}> {
 
 const styles = StyleSheet.create({
   buttonContainer: {
```

Comment:
nit: If these are blank now, I'd prefer deleting them completely so the code doesn't have a bunch of dead references in it